pkg/dependenciesdistributor/dependencies_distributor.go

```diff
@@ -687,8 +683,6 @@ func buildAttachedBinding(independentBinding *workv1alpha2.ResourceBinding, obje
 		Clusters:  independentBinding.Spec.Clusters,
 	})
 
-	bindingID := util.GetLabelValue(independentBinding.Labels, workv1alpha2.ResourceBindingPermanentIDLabel)
-	dependedLabels = util.DedupeAndMergeLabels(dependedLabels, map[string]string{dependedIDLabelKey: bindingID})
```

Comment:
Why was this `dependedIDLabelKey` label removed?

Reply:
Currently, this ID is not used and may be overwritten. To prevent more incorrect use, delete this ID.

Reply:
I think this can be submitted as a separate PR.

Then clearly write in the release-notes that this label needs to be removed.

Reply:
Good idea!